applied_computer_science_discord_bot/message_handlers/print_handler.py

```diff
@@ -0,0 +1,12 @@
+from discord import Message
+
+import applied_computer_science_discord_bot.message_handlers.virtual_message_handler as virtual_handler
+
+
+class PrintHandler(virtual_handler.CommandHandler):
+
+    async def handle(self, message:  Message, parsed_message):
+        if parsed_message.name == "print":
+            if "text" in parsed_message.arguments:
+                response = parsed_message.arguments["text"]
+            await message.channel.send(response)
```

Comment:
This will error out if "text" isn't in parsed_message.arguments. To fix it, you gave to declare the response variable outside of the ``` if "text" in  parsed_message.arguments ```